### ISSUE TITLE
[receiver/elasticsearch]: `_cluster/stats/_all` is not a valid path

### ISSUE THREAD

**Issue by davidji99** [open-telemetry/opentelemetry-collector-contrib#17867]
### Component(s)

receiver/elasticsearch

## Description

https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/16346 introduced this ES endpoint for querying metrics: `_cluster/stats/all`. According to ES 7 and 8 documentation, this is not a valid endpoint. The correct endpoint would be `/_cluster/stats/nodes/_all`.

## Steps to Reproduce

1. Run ES locally via docker
2. Query http://localhost:9200/_cluster/stats/_all.

## Expected Result

I should see a successful JSON response.

## Actual Result

I get this response:

```
{
"error": {
"root_cause": [
{
"type": "index_not_found_exception",
"reason": "no such index [_cluster]",
"resource.type": "index_expression",
"resource.id": "_cluster",
"index_uuid": "_na_",
"index": "_cluster"
}
],
"type": "index_not_found_exception",
"reason": "no such index [_cluster]",
"resource.type": "index_expression",
"resource.id": "_cluster",
"index_uuid": "_na_",
"index": "_cluster"
},
"status": 404
}
```

### Collector version

v0.69.0

## Environment
OS: (e.g., "Ubuntu 20.04")
Compiler(if manually compiled): (e.g., "go 14.2")

**Comment by junhuangli** [comment]
@djaglowski the fixes was checked in. Should we close this bug? Or we need to wait till the release.

**Comment by djaglowski** [comment]
Thanks again @junhuangli.

Closing this as resolved by #19414
